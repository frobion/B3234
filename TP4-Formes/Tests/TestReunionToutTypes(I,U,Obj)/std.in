R Rectangle1 1 3 6 0
S Segment1 0 0 1 1
PC PolygoneConvex1 1 1 3 -5 -2 -1
OR Reunion1 Rectangle1 Segment1
OI Intersection1 Rectangle1 Segment1
OR Reunion2 Reunion1 Intersection1 PolygoneConvex1
LIST
EXIT
