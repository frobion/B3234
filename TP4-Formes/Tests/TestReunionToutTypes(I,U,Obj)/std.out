OK
OK
OK
OK
OK
OK
R _Rectangle1 1 3 6 0
S _Segment1 0 0 1 1
OI Intersection1 _Rectangle1 _Segment1
DELETE _Rectangle1 _Segment1
PC PolygoneConvex1 1 1 3 -5 -2 -1
R Rectangle1 1 3 6 0
R _Rectangle1 1 3 6 0
S _Segment1 0 0 1 1
OR Reunion1 _Rectangle1 _Segment1
DELETE _Rectangle1 _Segment1
R _Rectangle1 1 3 6 0
S _Segment1 0 0 1 1
OR _Reunion1 _Rectangle1 _Segment1
DELETE _Rectangle1 _Segment1
R _Rectangle1 1 3 6 0
S _Segment1 0 0 1 1
OI _Intersection1 _Rectangle1 _Segment1
DELETE _Rectangle1 _Segment1
PC _PolygoneConvex1 1 1 3 -5 -2 -1
OR Reunion2 _Reunion1 _Intersection1 _PolygoneConvex1
DELETE _Reunion1 _Intersection1 _PolygoneConvex1
S Segment1 0 0 1 1


